i18n/README.pt-br.md

```diff
@@ -1,82 +1,83 @@
 <p align="center">
-<img width="300" src="https://raw.githubusercontent.com/supabase/supabase/master/packages/common/assets/images/supabase-logo-wordmark--light.svg#gh-light-mode-only">
-<img width="300" src="https://raw.githubusercontent.com/supabase/supabase/master/packages/common/assets/images/supabase-logo-wordmark--dark.svg#gh-dark-mode-only">
+<img src="https://user-images.githubusercontent.com/8291514/213727234-cda046d6-28c6-491a-b284-b86c5cede25d.png#gh-light-mode-only">
+<img src="https://user-images.githubusercontent.com/8291514/213727225-56186826-bee8-43b5-9b15-86e839d89393.png#gh-dark-mode-only">
 </p>
 
 ---
 
 # Supabase
 
-[Supabase](https://supabase.com) é uma alternativa de código aberto Firebase. Estamos construindo as características do Firebase usando ferramentas de código aberto de nível empresarial.
+o [Supabase] (https://supabase.com) é uma alternativa de código aberto ao Firebase. Estamos desenvolvendo os recursos do Firebase usando ferramentas de código aberto de nível empresarial.
 
-- [x] Banco de dados hospedado de Postgres. [Docs](https://supabase.com/docs/guides/database)
-- [x] Autenticação e Autorização. [Docs](https://supabase.com/docs/guides/auth)
-- [x] APIs auto-geradas.
+- [Banco de dados Postgres hospedado. [Docs](https://supabase.com/docs/guides/database)
+- [x] Autenticação e autorização. [Docs](https://supabase.com/docs/guides/auth)
+- [x] APIs geradas automaticamente.
```

Comment:
Hey @lyqht - something might have gone wrong with the first check box here (partially removed). Any ideas?